tools/design-tokens/build/BUILD.bazel

```diff
@@ -21,6 +21,7 @@ ts_library(
         "@npm//theo",
         "@npm//yaml",
         "@npm//yargs",
+        "@npm//@types/yargs",
```

Comment:
If you use the types then please install it as devDependency in the correct version @rowa-audil and @bardosmisi.
![image](https://user-images.githubusercontent.com/11156362/95328969-8b530800-08a6-11eb-8218-f195786486fa.png)